test/headcount_analyst_test.rb

```diff
@@ -6,23 +6,18 @@ class HeadcountAnalystTest < Minitest::Test
 
   def setup
     @dr = DistrictRepository.new
-  end
-
-  def test_can_create_an_instance
-    ha = HeadcountAnalyst.new(dr)
-
-    assert_instance_of HeadcountAnalyst, ha
+    @dr.load_data({
+      enrollment: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.